### ISSUE TITLE
[med] Chrome + Jaws: Role is not defined for invite button. 

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8481]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Navigate to Manage members using Tab key and press using Enter to select it.
5. Navigate to Invite button using Tab key and press using Enter to select it.
6. Invite new member window will appear.
7. Navigate to invite control using Tab key.
8. Observe the announcement of the screen reader.

## Expected Result:
Role should be defined for invite control, so that user can easily identify the function of the control.

## Actual Result:
Role is not defined for invite control. Screen reader only reading "Invite".

## Workaround:
Yes

## Area issue was found in:
Invite New Member

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader user might be confused without knowing the role of the interactive element, so the user is not able to take any action over the control.

## Suggested resolution:
Add the role="button" for the invite control or implement the element as a native button, so that the role can be announced by screen reader.                                                                                                                             Refer to:                                                                                                          https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role   

## Platform:
- Web
- iOS
- Desktop App
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/161671042-b5696095-4321-4b70-b089-f20c981caf9b.mp4
